src/targets/services/timeseriesWithoutAggregateMigration.js

```diff
@@ -28,24 +32,27 @@ const migrateTimeSeriesWithoutAggregation = async () => {
     settingsQuery.options
   )
   const appSetting = settings?.[0] || {}
-  const resp = await client.query(timeseriesWithoutAggregationQueryDef)
+  const resp = await client.queryAll(timeseriesWithoutAggregationQueryDef)
```

Comment:
This is a bit dangerous as it might reach the service execution time limit. But let's see if we receive alerts about it